README.rst

```diff
@@ -176,46 +195,179 @@ For flashing the board OpenOCD must be used. You can optionally use gdb
 as a frontend for OpenOCD as described below.
 
 You must first flash a bootstrap rom before flashing an application.
-Assuming the toolchain was unpacked into *$HOME/issm_2016.0.019/*, this can be
+Assuming the toolchain was unpacked into *$HOME/issm_2016/*, this can be
 done with:
 
-``$ cd $HOME/issm_2016.0.019/tools/debugger/openocd``
+``$ cd $HOME/issm_2016/tools/debugger/openocd``
+
+For D2000 start OpenOCD with the following command:
 
 ``$ ./bin/openocd -f scripts/board/quark_d2000_onboard.cfg``
 
+For SE C1000 start OpenOCD with the following command:
+
+``$ ./bin/openocd -f scripts/board/quark_se_onboard.cfg``
+
+Create a new terminal session at this point and set environment variables accordingly.
+Then launch a GDB session using:
+
 ``$ gdb``
 
-``(gdb) target remote :3333``
+To connect to the repote port, enter the following GDB command:
+
+``(gdb) target remote :333X``
+
+For D2000 and SE C1000 (Lakemont), the remote port value is 3333.
+For SE C1000 (ARC), the remote port value is 3334.
 
 ``(gdb) monitor clk32M 5000``
 
-``(gdb) monitor load_image $PATH_TO_QMSI/build/debug/quark_d2000/rom/quark_d2000_rom.bin 0x0``
+For D2000, the following commands are used to flash a ROM and application to the device:
+
+``(gdb) monitor load_image $PATH_TO_QMSI/build/release/quark_d2000/rom/quark_d2000_rom.bin 0x0``
+
+``(gdb) monitor load_image $PATH_TO_QMSI/examples/hello_world/release/quark_d2000/x86/bin/hello_world.bin 0x00180000``
+
+For SE C1000, the following commands are used to flash a ROM and application to the device:
+
+``(gdb) monitor load_image $PATH_TO_QMSI/build/release/quark_se/rom/quark_se_rom.bin 0xFFFFE000``
+
+Applications for the Lakemont core are flashed using the following command:
 
-``(gdb) monitor load_image $PATH_TO_QMSI/examples/hello_world/debug/quark_d2000/bin/hello_world.bin 0x00180000``
+``(gdb) monitor load_image $PATH_TO_QMSI/examples/hello_world/release/quark_se/sensor/bin/hello_world.bin 0x40000000``
+
+Applications for the ARC are flashed using the following command:
+
+``(gdb) monitor load_image $PATH_TO_QMSI/examples/hello_world/release/quark_se/x86/bin/hello_world.bin 0x40030000``
 
 Serial Output
 =============
 
 You can check UART console output with picocom or screen:
 
-``$ picocom -b 115200 --imap lfcrlf /dev/ttyUSB0``
+``$ picocom -b 115200 --imap lfcrlf /dev/ttyUSBXXX``
 
 or
 
-``$ screen /dev/ttyUSB0 115200``
+``$ screen /dev/ttyUSBXXX 115200``
+
+Where /dev/ttyUSBXXX is the path to the attached UART device.
+e.g. /dev/ttyUSB0
 
 
 Known Issues and Workarounds
```

Comment:
Let's make it clear to which version we refer to here. What about something like: "Known Issues and Workarounds - QMSI Release 1.1.0"? Or even "Known Issues and Workarounds" <NEW_LINE> "Affected version: QMSI 1.1.0" ?